test/test_collector.py

```diff
@@ -989,6 +989,82 @@ def test_collector_output_keys(collector_class, init_random_frames, explicit_spe
     del collector
 
 
+@pytest.mark.parametrize(
```

Comment:
Does it make sense to test on cpu only? Maybe we can skip it in that case

Reply:
ok yes I can remove it in that case